src/Ubiquity.NET.Llvm.Tests/TestObjectFile.cs

```diff
@@ -72,13 +72,15 @@ public static void Initialize( TestContext ctx )
             tm.EmitToFile( module, TestObjFileName, CodeGenFileType.ObjectFile );
         }
 
+        [TestCategory("ObjFiles")]
```

Comment:
If all of these tests need to be skipped on some target, then something is either fundamentally wrong with the tests, or the object file support. so a bug should be filed for this as disabling these all for a platform is cutting off a big chunk of functionality of the library.